### PR TITLE
navitia-proto: attributs added for ODT

### PR DESCRIPTION
**Pull request by kadhikari**
Attributs added are to be used in a new api "odt_stop_points".
This api returns all the stop_points in a zone ODT for a coordinate.

### CODE REVIEW

File: response.proto [hunk 1]
```diff
@@ -492,11 +492,18 @@ message Response{
 
      // Car CO2 emission
      optional Co2Emission car_co2_emission = 70;
-     
+
      optional StreetNetworkRoutingMatrix sn_routing_matrix = 72;
+
+     //experimental
+	 repeated ODTStopPoint odt_stop_points = 73;
```
Comment (pbougue) [inline response.proto]: indent is weird

File: response.proto [hunk 2]
```diff
@@ -492,11 +492,18 @@ message Response{
...
 }
 
 message NearestStopPoint{
     optional StopPoint stop_point = 1;
     optional int32 access_duration = 2;
 }
+
+message ODTStopPoint {
+	optional StopPoint stop_point = 1;
```
Comment (pbougue) [inline response.proto]: indent

**Comment by kadhikari** [comment]
@pbougue i modified on your remarks

File: response.proto [hunk 3]
```diff
@@ -481,19 +481,22 @@ message Response{
...
-     optional Co2Emission car_co2_emission = 70;
-     
-     optional StreetNetworkRoutingMatrix sn_routing_matrix = 72;
+    // Car CO2 emission
+    optional Co2Emission car_co2_emission = 70;
+
+    optional StreetNetworkRoutingMatrix sn_routing_matrix = 72;
+
+    //experimental
+    repeated StopPoint odt_stop_points = 73;
```
Comment (TeXitoi) [inline response.proto]: Not needed.

**Pull request merged**